libs/deal-ticket/src/components/deal-ticket/deal-ticket.spec.tsx

```diff
@@ -654,4 +686,40 @@ describe('DealTicket', () => {
       new Date(screen.getByTestId<HTMLInputElement>(datePicker).value).getTime()
     ).toEqual(now);
   });
+
+  describe('market states not accepting orders', () => {
+    const states = [
+      Schema.MarketState.STATE_REJECTED,
+      Schema.MarketState.STATE_CANCELLED,
+      Schema.MarketState.STATE_CLOSED,
+      Schema.MarketState.STATE_SETTLED,
+      Schema.MarketState.STATE_TRADING_TERMINATED,
+    ];
+
+    states.forEach((marketState) => {
+      describe(`Market State: ${marketState}`, () => {
+        beforeEach(() => {
```

Comment:
`beforeEach` and and `it`'s are not needed, they can be removed and you can leave only what they executes